podium-gateway/src/main/webapp/app/request/main-detail/detail/request-detail.component.ts

```diff
@@ -34,29 +35,35 @@ import { DeliveryService } from '../../../shared/delivery/delivery.service';
 
 export class RequestDetailComponent implements OnDestroy {
 
+    public RequestReviewDecision: typeof RequestReviewDecision = RequestReviewDecision;
```

Comment:
needed to make RequestDetailComponent .RequestReviewDecision as RequestReviewDecision instance not only type